WordPress/Classes/Services/PostCoordinator.swift

```diff
@@ -261,6 +266,15 @@ class PostCoordinator: NSObject {
         topViewController.present(alert, animated: true)
     }
 
+    private func trackError(_ error: Error, operation: String) {
+        DDLogError("post-coordinator-\(operation)-failed: \(error)")
+
+        if let error = error as? TrackableErrorProtocol, var userInfo = error.getTrackingUserInfo() {
+            userInfo["operation"] = operation
+            WPAnalytics.track(.postCoordinatorErrorEncountered, properties: userInfo)
```

Comment:
Just wanted to note that for Tracks, snake case is preferred. But this is a property value so I don't have strong feelings - I'll let you decide!

Reply:
Oh, for property names. Totally – updated.